compiler/passes/scopeResolve.cpp

```diff
@@ -1956,9 +1956,42 @@ bool lookupThisScopeAndUses(const char*           name,
                             BaseAST*              context,
                             BaseAST*              scope,
                             std::vector<Symbol*>& symbols,
+                            bool skipExternBlocks,
                             std::map<Symbol*, astlocT*>& renameLocs,
                             bool storeRenames,
                             std::map<Symbol*, VisibilityStmt*>& reexportPts) {
+
+
+  ModuleSymbol* scopeIsModule = nullptr;
```

Comment:
This is named like it's a boolean, but it's pulling double duty as part of the module cache.  I think I'd rather have a separate variable for the check and for the module symbol itself - this will make it obvious (without having to check the declaration) that modSymsCache is storing a module and the name rather than storing a boolean for some reason.